packages/node-experimental/src/utils/contextData.ts

```diff
@@ -18,5 +21,17 @@ export function getScopesFromContext(context: Context): CurrentScopes | undefine
  * This will return a forked context with the Propagation Context set.
  */
 export function setScopesOnContext(context: Context, scopes: CurrentScopes): Context {
+  // So we can look up the context from the scope later
+  SCOPE_CONTEXT_MAP.set(scopes.scope, context);
+  SCOPE_CONTEXT_MAP.set(scopes.isolationScope, context);
+
   return context.setValue(SENTRY_SCOPES_CONTEXT_KEY, scopes);
 }
+
+/**
+ * Get the context related to a scope.
+ * TODO v8: Use this for the `trace` functions.
```

Comment:
These functions still live in `opentelemetry` which needs to be aligned for v8. It's not worth it to do that now IMHO, just wanted to show that it is possible to implement this in the new model, which it is!